lib/protobuf/transform_module.ex

```diff
@@ -50,4 +60,13 @@ defmodule Protobuf.TransformModule do
   Called after a message is decoded.
   """
   @callback decode(message(), type()) :: value()
+
+  @doc """
+  Transforms the typespec for modules using this transformer
+
+  Optional. If this callback is not present, the default typespec will be used.
```

Comment:
ExDoc already marks this as optional thanks to `@optional_callbacks`

```suggestion
  If this callback is not present, the default typespec will be used.
```